### PR TITLE
Modified run_EE_tests.sh to allow for custom server URL

### LINKED ISSUE #213: Modify rh-che PR chceck to run tests against deployed server

### PR DESCRIPTION
**Pull request by ScrewTSW**
Closes #213

### CODE REVIEW

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
@rhopp @Katka92 ready for review.

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
[test]

**Comment by ScrewTSW** [comment]
[test]

File: cico/run_EE_tests.sh [hunk 1]
```diff
@@ -15,15 +17,22 @@ if [[ -z "${OSO_NAMESPACE}" ]]; then
...
 fi
 
 cd /home/fabric8/che
 export DISPLAY=:99
 mvn clean install -DskipTests
-mvn clean verify -f tests/pom.xml -DosioUrlPart=$OSIO_URL_PART -Dtest=$TEST_SUITE -DopenShiftMasterURL=$OSO_MASTER_URL -DkeycloakToken=$KEYCLOAK_TOKEN -DopenShiftNamespace=$OSO_NAMESPACE -DosioUsername=$OSIO_USERNAME -DosioPassword=$OSIO_PASSWORD
+if [[ -z "${CUSTOM_CHE_SERVER_FULL_URL}" ]]; then
+  mvn clean verify -f tests/pom.xml -DosioUrlPart=$OSIO_URL_PART -Dtest=$TEST_SUITE -DopenShiftMasterURL=$OSO_MASTER_URL -DkeycloakToken=$KEYCLOAK_TOKEN -DopenShiftNamespace=$OSO_NAMESPACE -DosioUsername=$OSIO_USERNAME -DosioPassword=$OSIO_PASSWORD
+else
+  echo "Username: ${OSIO_USERNAME:0:1}***${OSIO_USERNAME:5:1}***${OSIO_USERNAME:8:2}***"
```
Comment (rhopp) [inline cico/run_EE_tests.sh]: I would prefer if these outputs weren't here. They were just for debugging one issue and they are not needed anymore.

File: tests/src/test/java/redhat/che/functional/tests/AbstractCheFunctionalTest.java [hunk 2]
```diff
@@ -109,6 +109,8 @@ private void openBrowser(CheWorkspace wkspc) {
             Graphene.waitGui().withTimeout(10, TimeUnit.SECONDS).until().element(usernameField).is().visible();
             login();
         } catch (WebDriverException e) {
+            LOG.error("Login failed");
+            e.printStackTrace();
```
Comment (rhopp) [inline tests/src/test/java/redhat/che/functional/tests/AbstractCheFunctionalTest.java]: What about "squashing" these two lines into one?
https://logging.apache.org/log4j/1.2/apidocs/org/apache/log4j/Category.html#info(java.lang.Object,%20java.lang.Throwable)

**Pull request merged**